Makefile

```diff
@@ -14,7 +14,8 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # $2 - Dockerfile path
 # $3 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+IMAGE_TAG?=registry.svc.ci.openshift.org/ocp/4.3:cluster-policy-controller
```

Comment:
Move those variables higher, that's the preferred approach, see https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/Makefile as an example.

Reply:
I see, updated